freebasics/templates/search/search_results.html

```diff
@@ -5,30 +5,21 @@
 {% if search_results %}
 <div class="block-search fb-body fb-card">
   <h6 class="block-heading fb-accent-1">Search Results</h6>
-  <h6 class="sub-heading">{{search_query}}</h6>
+  <h6 class="sub-heading">Search Results for {{search_query}}</h6>
```

Comment:
You should put these words in `{% trans "Search results for" %}` (same for seach results above)